components/writers/database/mssql/index.md

```diff
@@ -75,3 +75,12 @@ freeze waiting for the locks to be released.
 
 Additionally, you can specify a **Primary key** of the table, a simple column **Data filter**, and a filter for
 [incremental processing](/storage/tables/#incremental-processing).
+
+## Binary types
+
+For binary types `binary`, `varbinary`, `image`, the following rules apply:
+- If value **starts with `0x`** and **length of the value is even number**, then it is a HEX value according to the [MsSQL specification (style 1)](https://docs.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?redirectedfrom=MSDN&view=sql-server-ver15#binary-styles).

```

Comment:
Opravovali sme vkladanie binarnych hodnot v `wr-mssql`.

Pred zmenou sa `image` typ nedal pouzit vobec, a HEX hodnoty sa ukladali, ako keby boli string. Aj ked ich mal klient v CSV v takom formate, ktory je pre MsSQL validny/bezny pre binarne hodnoty.

Kedze je tam podmienka zo strany MsSQL, ze dlzka HEX hodnoty musi byt parna, tak to davam radsej aj do docs.